.pre-commit-config.yaml

```diff
@@ -3,3 +3,4 @@ repos:
     rev: 19.3b0
     hooks:
       - id: black
+        args: [--line-length 100]
```

Comment:
👏

Reply:
Standing ovation actually, and what about that rev? we don't have rev 19.3, but 19.10 something if I remember correctly

Reply:
Right, well spotted!